modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/SliderWithLabel.es.js

```diff
@@ -30,6 +31,7 @@ function SliderWithLabel({
 			<div className="flex-row form-group-item">
 				<ClaySlider
 					className="w-100"
+					disabled={selectedTestType === 'MAB'}
```

Comment:
Use disabled prop instead

Reply:
@interaminense you mean passing only `disabled` here? With no condition?

Reply:
done